.Rproj.user/B5F94F77/sources/s-C9E6AA8B/83198F1F-contents

```diff
@@ -29,19 +29,45 @@
 #' @author \href{https://dominiquemakowski.github.io/}{Dominique Makowski}
 #'
 #' @export
-rope <- function(posterior, bounds = c(-0.1, 0.1), CI = 90, verbose = TRUE) {
+rope <- function(posterior, bounds = "default", CI = 90, verbose = TRUE) {
   UseMethod("rope")
 }
 
 
+#' @method as.numeric rope
+#' @export
+as.numeric.rope <- function(x, ...){
```

Comment:
this doens't work 😭

Reply:
I just downloaded and played around with the dev-branch, and I am confused as well. What I've read from `?as.numeric`:

> as.numeric is a generic function, but S3 methods must be written for as.double. It is identical to as.double.

So I tried an `as.double.rope`,  but had no success, either...

Reply:
Ok, should work now, with `as.double.rope()` - in the last try, I forgot to update the NAMESPACE. ^^